src/domains/movie.domain.js

```diff
@@ -0,0 +1,147 @@
+const prisma = require("../utils/prisma");
+const Types = require("../utils/types.d");
+
+/**
+ * @param {Number} [runTimeLessThan]
+ * @param {Number} [runTimeGreaterThan]
+ * @returns {Promise<Types.Movie[]>}
+ */
+async function selectMovies(runTimeLessThan, runTimeGreaterThan) {
+  const runtimeWhere = {};
+
+  if (typeof runTimeLessThan === "number") {
+    runtimeWhere.lt = runTimeLessThan;
+  }
+
+  if (typeof runTimeGreaterThan === "number") {
+    runtimeWhere.gt = runTimeGreaterThan;
+  }
+
+  return await prisma.movie.findMany({
+    where: {
+      runtimeMins: runtimeWhere,
```

Comment:
I don't quite understand how this gets interpreted with the "smaller than (runtimeWhere.lt) and greater then (runtimeWhere.gt)" – runtimeWhere is in the state of { gt: X, lt: Y }, correct? does Prisma have some automatic interpretation of this? I see you only pass numbers as params, so it's not like you have an "<", ">" in there.